src/shared/mount-util.c

```diff
@@ -772,6 +774,29 @@ int bind_mount_in_namespace(
         assert(src);
         assert(dest);
 
+        r = namespace_open(target, NULL, &mntns_fd, NULL, NULL, &root_fd);
+        if (r < 0)
+                return r;
+
+        r = fd_get_path(mntns_fd, &mntns);
+        if (r < 0)
+                return r;
+
+        r = namespace_open(getpid_cached(), NULL, &self_mntns_fd, NULL, NULL, NULL);
```

Comment:
specify "0" as first arg. it's the same and a tiny bit more efficient

Reply:
TIL - fixed